webapp/sources/rudder/rudder-core/src/test/scala/com/normation/rudder/services/queries/TestQueryProcessor.scala

```diff
@@ -857,6 +857,8 @@ class TestQueryProcessor extends Loggable {
       ] }
       """ :: Nil
 
+
```

Comment:
Do you wanted to add those lines ?

Reply:
Does it matter?

Reply:
no :) just jealous of your github stat increase ;)